hera_cal/smooth_cal.py

```diff
@@ -79,34 +79,32 @@ def freq_filter(gains, wgts, freqs, filter_scale=10.0, tol=1e-09, window='tukey'
         filter_scale: frequency scale in MHz to use for the low-pass filter. filter_scale^-1 corresponds
             to the half-width (i.e. the width of the positive part) of the region in fourier
             space, symmetric about 0, that is filtered out.
-        tol: CLEAN algorithm convergence tolerance (see aipy.deconv.clean)
-        window: window function for filtering applied to the frequency axis.
-            See aipy.dsp.gen_window for options.
         skip_wgt: skips filtering rows with very low total weight (unflagged fraction ~< skip_wgt).
             filtered is left unchanged and info is {'skipped': True} for that time.
             Only works properly when all weights are all between 0 and 1.
-        maxiter: Maximum number of iterations for aipy.deconv.clean to converge.
-        win_kwargs : any keyword arguments for the window function selection in aipy.dsp.gen_window.
+        mode: deconvolution method to use. See uvtools.dspec.fourier_filter for full list of supported modes.
+              examples include 'dpss_leastsq', 'clean'.
+        filter_kwargs : any keyword arguments for the window function selection in aipy.dsp.gen_window.
```

Comment:
This isn't true anymore, right? It's much more general now...

Reply:
That's true. I've updated the docstring to reflect this properly.